conf/kubernetes-shuffle-service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: spark-shuffle-service
+    version: 2.1.0
+  name: shuffle
+spec:
+  template:
+    metadata:
+      labels:
+        app: spark-shuffle-service
+    spec:
+      volumes:
+        - name: temp-volume
+          hostPath:
+            path: '/tmp'
```

Comment:
I don't think it would be a helm chart since it doesn't deploy an entire application; just a dependent component (and the whole overhead of helm/tiller would be prohibitive). This is only a sample YAML that a user might use. I was thinking of distributing a YAML file that the user may modify according to their needs.